test/controller/user.js

```diff
@@ -32,6 +32,40 @@ describe('UserController', () => {
     expect(res.header.location).to.equal('/api/v1/user/usertester');
   });
 
+  describe.only('forgotten username', () => {
```

Comment:
Remove `describe.only`.

(I've been meaning to automatically lint for this with [`no-restricted-properties`](http://eslint.org/docs/rules/no-restricted-properties), but haven't gotten around to it.)

```yaml
rules:
  no-restricted-properties: [error, {object: describe, property: only}, {object: it, property: only}]
```